api/resid/gvk.go

```diff
@@ -122,6 +122,8 @@ func (x Gvk) Equals(o Gvk) bool {
 // Namespace should be first.
 // In some cases order just specified to provide determinism.
 var orderFirst = []string{
+	"MutatingWebhookConfiguration",
+	"ValidatingWebhookConfiguration",
```

Comment:
Why before Namespace?

Reply:
Missed the comment ontop. The general notion is that a Mutating/Validating webhook can mutate/validate any of the next resource kinds, so they should be first because of that.